scripts/prism_4km_monthly_modern.py

```diff
@@ -1,9 +1,12 @@
 #retriever
 
 """Retriever script for direct download of PRISM climate data"""
+from future import standard_library
+standard_library.install_aliases()
+from builtins import range
 
 from retriever.lib.templates import Script
-import urlparse
+import urllib.parse
```

Comment:
Python 2 has `urlparse` and Python 3 has `urllib.parse`. Similar to that I explained above.